pooch/__init__.py

```diff
@@ -6,6 +6,11 @@
 from .downloaders import HTTPDownloader, FTPDownloader
 from .processors import Unzip, Untar, Decompress
 
+try:
+    from .downloaders import SFTPDownloader
+except ImportError:
+    pass
```

Comment:
@luke-gregor it's probably best to always have the class defined in the API but have it raise an exception if `paramiko` is missing. This avoids the need for separate documentation `requirements.txt`.

Reply:
Ah, yes! Way more elegant :)